package.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "type": "module",
   "scripts": {
-    "start": "pm2 start index.js --name ngmp-network-app",
-    "stop": "pm2 stop ngmp-network-app"
+    "start": "tsc && pm2 start dist/module_6/index.js --name ngmp-network-app",
```

Comment:
As recommendation, it is a good practice to clean/remove the `dist` or compiled directory to have a clean application.

Not changes requested